src/components/dataset/details/DatasetDetailView.jsx

```diff
@@ -32,20 +39,58 @@ const styles = (theme) => ({
   mainColumn: {
     display: 'flex',
     flexDirection: 'column',
-    paddingLeft: '0.5rem',
+    marginLeft: 40,
```

Comment:
dont we have themes for these numbers so we stay consistent?

Reply:
for some of these numbers yes! i've been using rem instead of spacing lately since that's a more general standard, but there are some places that we don't want to scale if we resize things so we keep using pixels